### PR TITLE
Added update method for  Email campaigns table

### LINKED ISSUE #8116: [Sendinblue] Implement UPDATE for the Email Campaigns table
MindsDB integrates with Sendinblue, which provides interfaces to connect to Sendinblue via APIs and pull repository data into MindsDB

## Resources

Familiarize yourself with the existing PayPal integration: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/sendinblue_handler

## Steps to Follow

Currently, the Sendinblue integration with MindsDB accesses the Email Campaigns Table.

The task is to extend [this implementation](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/sendinblue_handler) to include the UPDATE command for the Email Campaigns table.

Please provide screenshots or a short video in the PR to show successful test results.

## :loudspeaker: Additional rewards :trophy:

Each integration extension issue brings 15+ points for SWAG and entry into the draw for a :point_down: :
-  :computer:  [Razer Blade 16 Laptop](https://www.razer.com/gaming-laptops/Razer-Blade-16/RZ09-0483SEJ3-R3U1)
- :shirt: :bear:  [MindsDB Swag](https://store.covver.io/mindsdb/collections/swag-collection)

:information_source: For more info check out https://mindsdb.com/hacktoberfest/ :point_left:

### PR DESCRIPTION
**Pull request by Sekhar-Kumar-Dash**
## Description

added update method for email campaigns table

Fixes #8116
- [x] ⚡ New feature (non-breaking change which adds functionality)
- [x] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

Video attachment:

https://github.com/mindsdb/mindsdb/assets/119131588/5d3e69fc-7c0f-43c5-8b46-e107a45c5a7c

### CODE REVIEW

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Please don't remove the default PR template when you submiting PR. Make sure you have all of the steps checked

**Comment by Sekhar-Kumar-Dash** [comment]
> Please don't remove the default PR template when you submiting PR. Make sure you have all of the steps checked



> Please don't remove the default PR template when you submiting PR. Make sure you have all of the steps checked

okay

**Comment by Sekhar-Kumar-Dash** [comment]
@ZoranPandovski  its done you review it now

**Comment by Sekhar-Kumar-Dash** [comment]
https://github.com/mindsdb/mindsdb/assets/119131588/586863a3-0e11-460e-ba34-3e600b32e9ce

**Comment by Sekhar-Kumar-Dash** [comment]
https://github.com/mindsdb/mindsdb/assets/119131588/55380b64-100d-4ecf-866f-26c165d76ff9

**Comment by Sekhar-Kumar-Dash** [comment]
Upload Method
https://github.com/mindsdb/mindsdb/assets/119131588/df10e0c6-e950-4f9b-9fe0-516775cdd1cd

File: mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py [hunk 1]
```diff
@@ -58,4 +60,46 @@ def get_email_campaigns(self, **kwargs):
...
+
+    def update_email_campaigns(self, campaign_ids: List[int], values_to_update: Dict) -> None:
+        connection = self.handler.connect()
+        email_campaigns_api_instance = sib_api_v3_sdk.EmailCampaignsApi(connection)
+
+        for campaign_id in campaign_ids:
+            try:
+                email_campaigns_api_instance.update_email_campaign(campaign_id, values_to_update)
+            except ApiException as e:
+                print(f"Exception when updating campaign {campaign_id}: {e}")
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py:105]: Lets log and raise the exception instead of printing

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py (hunk 1 above)
Comment (Sekhar-Kumar-Dash) [reply mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py:105]: okay

Comment (Sekhar-Kumar-Dash) [reply mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py]: @ZoranPandovski  made the changes  please review it

**Comment by lnxpy** [comment]
Hi @Sekhar-Kumar-Dash would you squash your commits into one so that our history would be cleaner then? :)
Thanks.

**Review by lnxpy — CHANGES_REQUESTED** [review]
It seems that you don't use `pre-commit`. Please install it and try squashing with all pre-commit hooks installed.

**Comment by lnxpy** [comment]
@Sekhar-Kumar-Dash Nothing left to review from my side. Those changes are still required. Thank you for your contribution! :D

**Comment by Sekhar-Kumar-Dash** [comment]
> @Sekhar-Kumar-Dash There is nothing left to review from my side. Those changes are still required. Thank you for your contribution! :D

can I do it  from  next time

**Comment by lnxpy** [comment]
That's fine. Take your time. Thanks again. ❤️

**Comment by Sekhar-Kumar-Dash** [comment]
> That's fine. Take your time. Thanks again. ❤️

Thank you

**Comment by paxcema** [comment]
Hey just chiming in, @Sekhar-Kumar-Dash please address @lnxpy's comments for _this_ PR in order to get it merged. Thanks for contributing!

**Comment by Sekhar-Kumar-Dash** [comment]
> Hey just chiming in, @Sekhar-Kumar-Dash please address @lnxpy's comments for _this_ PR in order to get it merged. Thanks for contributing!

hey @paxcema   @inxpy told me that all  commit will be combined together so no need for that

**Comment by lnxpy** [comment]
LGTM :D

**Review by chandrevdw31 — APPROVED** [review]

**Review by tcchawla — APPROVED** [review]
@Sekhar-Kumar-Dash, Looks Perfect!

**Pull request merged**